src/wb_ext_dnd.ts

```diff
@@ -258,14 +258,15 @@ export class DndExtension extends WunderbaumExtension<DndOptionsType> {
 
       // --- drag ---
     } else if (e.type === "drag") {
-      // This event occurs very often...
+      if (dndOpts.drag) srcNode._callEvent("dnd.dragDrag", { event: e });
```

Comment:
Here we another `dragDrag` (I tried to fix it in your workOnDD branch, but it didn't appear here)

Reply:
think it was only a github UI problem. But I rebased and force pushed now.